package.json

```diff
@@ -12,16 +12,21 @@
     "coverage": "jest --coverage"
   },
   "dependencies": {
-    "bootstrap": "^5.0.1",
+    "@babel/runtime-corejs3": "^7.14.7",
```

Comment:
@DianaVargasOrellana I don't think we will need this plugin since we are using TypeScript, I think this plugin is used to have the Ecma6 built-ins like Promise, Map, etc. in a javascript environment, but, we are using TypeScript, so those built-ins are managed the TS compiler.